packages/astro/src/core/errors/errors-data.ts

```diff
@@ -147,8 +144,8 @@ export const MissingMediaQueryDirective = {
  * @docs
  * @message Unable to render `COMPONENT_NAME`. There are `RENDERER_COUNT` renderer(s) configured in your `astro.config.mjs` file, but none were able to server-side render `COMPONENT_NAME`.
  * @see
- * - [Frameworks components](https://docs.astro.build/en/core-concepts/framework-components/)
- * - [UI Frameworks](https://docs.astro.build/en/guides/integrations-guide/#official-integrations)
+ * - [Configuring a UI framework integration](https://docs.astro.build/en/core-concepts/framework-components/)
+ * - [List of all official UI framework integrations](https://docs.astro.build/en/guides/integrations-guide/#official-integrations)
```

Comment:
Those links (and all the other @see when possible) kinda purposely matched the headers they're linking to, so that there's a mental link between where you clicked vs where you landed.

(The second link is okay, since it just links to a list and there's no text to read)

Reply:
I'm okay with either! I'll revert if that's your preference.